pages/index.js

```diff
@@ -0,0 +1,17 @@
+import { Fragment } from 'react';
+
+
+export default function Home() {
+  return (
+      <Fragment>
```

Comment:
Is there a difference between using "Fragment" and just using <>

Reply:
They are the same thing sir, you can check the react docs for more confirmation, <Fragment> Is more descriptive. that's why I am using it unlike <> which both also returns an extra div

Reply:
Okay thanks for clearing that up